typings/react-dropzone.d.ts

```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 
+export { FileWithPath } from "file-selector";
```

Comment:
I'm not so sure about this 🤔 Ppl could import it from the file-selector lib as it's already a dependency of this lib (I think).

Reply:
Technically, consumers _could_ import `FileWithPath` from `file-selector`... however, there'd be no safe guard for us from removing the `file-selector` dependency from future minor/patch versions of `react-dropzone` and breaking said consumers.

This leaves us with two options:

1. Export the type from `react-dropzone` (as done here)
2. Move the `file-selector` dep to a `peerDependency` and `devDependency`

I much prefer the first as it less work for consumers of the library to get setup and peer deps can be confusing 🙂